### DISCUSSION TITLE
Using Unicorn on whole website

### DISCUSSION THREAD

**Discussion by gschurck** [django-commons/django-unicorn#435] [Q&A]
Is it possible to create a website entirely with Unicorn without refreshing the page ?
Like if the navigation bar is a component and when clicking it changes the parent subview component to go from one page to another, without refreshing the whole page ?
I don't know if it's possible to chose which component view to load in the Python component instead of in the template, to make it a public variable that can be modified ?

**Comment by martindemello** [comment]
I would love to hear some strategies for this too. When trying it I ran into https://github.com/adamghill/django-unicorn/issues/437 but I'm not sure if I'm just doing it wrong or it's fundamentally not something unicorn supports.
